docs/DeveloperGuide.md

```diff
@@ -312,49 +307,69 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
   Use case ends.
 
-* 3a. The given index is invalid.
+**Use case: UC02 - Delete a profile**
+
+**MSS**
+
+1. User requests to <u>view all profiles (UC01)</u>.
+2. User requests to delete a specific profile in the list.
+3. NUScheduler deletes the person.
 
-    * 3a1. NUScheduler shows an error message.
+    Use case ends.
 
-      Use case resumes at step 2.
+**Extensions**
+
+* 2a. The given index is invalid.
+
+    * 2a1. NUScheduler shows an error message.
+
+      Use case ends.
```

Comment:
Should it be `Use case resumes at step 1` instead?

Reply:
wouldn't resuming at step 1 be the same as the use case ending?

Reply:
I think I made a mistake for this edit. The original was to resume from step 2. I think this was meant to signify that user can try to request deleting profile again but does not have to go through step 1 (List all profiles to check for index). I will revert this section?

Reply:
I feel like there should be some way to show that the user can just try to request deleting again and again until it's successful, that's why I think `Use case ends.` may not be very suitable. But I do agree that saying resuming at step 1 sounds a little odd.